src/BlockTypes/AbstractBlock.php

```diff
@@ -151,12 +151,12 @@ protected function enqueue_scripts( array $attributes = [] ) {
 	 */
 	protected function get_skeleton_inline_script() {
 		return "<script>
-			const containers = document.querySelectorAll( 'div.wc-block-skeleton' );
+			var containers = document.querySelectorAll( 'div.wc-block-skeleton' );
 
 			if ( containers.length ) {
 				Array.prototype.forEach.call( containers, function( el, i ) {
-					const w = el.offsetWidth;
-					let classname = '';
+					var w = el.offsetWidth;
+					var classname = '';
```

Comment:
Actually here it would be fine using `const` and `let` because they are scoped inside the inner function. But having it as `var` makes sense as well to keep it consistent.

Reply:
Yeah that's why I changed it. I didn't want to mix const/let with old vars.